typescript/web/src/connectors/labeling-state/index.ts

```diff
@@ -162,20 +152,25 @@ export const useLabelingStore = create<LabelingState>(
     showLabelsGeometry: true,
     showLabelsName: true,
     toggleViewMode: () => {
+      // Change labels visibility (geometry and name).
```

Comment:
Putting the comment only here will avoid it to be discoverable in the editor when used.

Reply:
Although I understand why you wanted to comment-out how it works, the thing is that:
* The implementation will evolve in a not-that-far future (we've already spoken about it during the last DSM) and if we put the comment on the type declaration, they are great chances that we'll forget to update it as we'll probably won't need to change the function typings (and won't read the comment it would had).
* It's not that important to know between which view mode it toggles at an upper-level in the code.
* If we want to have more details about what happens in one of our own functions, we're used to navigate to its implementation ; where I've moved the details you've provided 😉